### ISSUE TITLE
Searching monomer library broken

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#88]
Hi Paul,

Searching the monomer library (in `main`, commit `a311017`) retrieves very few results. The chemical structures also look weird, but I suspect that is due to longstanding and previously reported issues for heme-like molecules in the CCP4 monomer library.

**Coot 1.06-pre:**
<img width="951" alt="image" src="https://github.com/pemsley/coot/assets/19766818/79c81ea3-f5d0-461f-8f52-b413989446c6">

**Coot 0.9.8.91:**
![image](https://github.com/pemsley/coot/assets/19766818/26417243-9b60-481b-9e11-f3887137dc09)

**Comment by pemsley** [comment]
The major problem here, I suspect, is that the reference monomer library is different.

It seems that CCP4 distro is more up to date than the homebrew distro.

**Comment by olibclarke** [comment]
I don't think so - because I can use "get monomer" to get the ones that don't appear, they just don't appear in the search results..
